run.sh

```diff
@@ -44,17 +44,30 @@ run_cal()
 	./tools/bl srccfg "$device" 5  1 0 "$oversample" 0 0 # 5.0V
 	./tools/bl srccfg "$device" 6  1 0 "$oversample" 0 0 # Temperature
 
+	# There are 19 channels including 16 LEDs plus 3.3V, 5.0V and Temperature
 	# chancfg <channel> <source> [offset] [shift] [sample32]
-	./tools/bl chancfg "$device" 0 0  0  0  1 # Photodiode 1
-	./tools/bl chancfg "$device" 1 1  0  0  1 # Photodiode 2
-	./tools/bl chancfg "$device" 2 2  0  0  1 # Photodiode 3
-	./tools/bl chancfg "$device" 3 3  0  0  1 # Photodiode 4
-	./tools/bl chancfg "$device" 4 4  0  0  1 # 3.3V
-	./tools/bl chancfg "$device" 5 5  0  0  1 # 5.0V
-	./tools/bl chancfg "$device" 6 6  0  0  1 # Temperature
+	./tools/bl chancfg "$device" 0  2  0  0  1 # Photodiode 3
+	./tools/bl chancfg "$device" 1  2  0  0  1 # Photodiode 3
+	./tools/bl chancfg "$device" 2  2  0  0  1 # Photodiode 3
+	./tools/bl chancfg "$device" 3  2  0  0  1 # Photodiode 3
+	./tools/bl chancfg "$device" 4  3  0  0  1 # Photodiode 4
+	./tools/bl chancfg "$device" 5  3  0  0  1 # Photodiode 4
+	./tools/bl chancfg "$device" 6  3  0  0  1 # Photodiode 4
+	./tools/bl chancfg "$device" 7  3  0  0  1 # Photodiode 4
+	./tools/bl chancfg "$device" 8  1  0  0  1 # Photodiode 2
+	./tools/bl chancfg "$device" 9  1  0  0  1 # Photodiode 2
+	./tools/bl chancfg "$device" 10 1  0  0  1 # Photodiode 2
+	./tools/bl chancfg "$device" 11 1  0  0  1 # Photodiode 2
+	./tools/bl chancfg "$device" 12 0  0  0  1 # Photodiode 1
+	./tools/bl chancfg "$device" 13 0  0  0  1 # Photodiode 1
+	./tools/bl chancfg "$device" 14 0  0  0  1 # Photodiode 1
+	./tools/bl chancfg "$device" 15 0  0  0  1 # Photodiode 1
+	./tools/bl chancfg "$device" 16 4  0  0  1 # 3.3V
+	./tools/bl chancfg "$device" 17 5  0  0  1 # 5.0V
+	./tools/bl chancfg "$device" 18 6  0  0  1 # Temperature
```

Comment:
This changes `cal` to operate in flash mode.  Also the same for `acq`.  I think we should leave them unchanged for doing experiments in "continuous mode", and add some new things like `flash-cal` and `flash-acq` for continuous mode.

Reply:
ok, will make the change.

btw. have you saw my `TODO` tag in run_acq? I've no idea how to get the shift values you put in there, so left that section unchanged.

Reply:
Replied to the TODO.

Reply:
I have kept this change, and introduced a environment variable to run.sh to choice between flash mode and continuous mode